src/utils/media-utils.js

```diff
@@ -615,10 +615,9 @@ export async function resolveMediaInfo(urlString) {
 
   // We want to resolve and proxy some hubs urls, like rooms and scene links,
   // but want to avoid proxying assets in order for this to work in dev environments
-  const isLocalModelAsset =
-    isNonCorsProxyDomain(url.hostname) && (guessContentType(url.href) || "").startsWith("model/gltf");
```

Comment:
@johnshaughnessy do you have any memory of why models were the only local assets that don't use a canonical URL? With the `newLoader` path - uploaded images, movies, and sounds are broken links when working in a local dev env.

I know [this line also appears in "components/media-loader.js"](https://github.com/mozilla/hubs/blob/7ac8f19db85dd4a13fb68fe2cd911d24bddb26ab/src/components/media-loader.js#LL367C1-L367C1) which dates back [to this PR.](https://github.com/mozilla/hubs/pull/3904)

 If I make the change shown here then all media tested (glb, mp3, mp4, jpg) work with `newLoader` in local dev and on my personal instance. That said,  I am unsure of the original intent in having non-model assets use canonical urls and don't want to introduce a regression.

Reply:
Asking you specifically because you were the last to work on this section. It was 9 months ago though so no worries if you don't remember.

Reply:
Hmm, unfortunately no. I don't know why we made an exception only for models. But, given that we're OK with the exception for models, I can't think of why we wouldn't be OK to lift the restriction for other media types like videos and images. 

At the risk of some regression, I'll go ahead with this PR (despite not understanding what lies on the other side of the [fence](https://en.wikipedia.org/wiki/G._K._Chesterton#Chesterton's_fence)).